### PR TITLE
Dolphins refactory

### PR DESCRIPTION
**Pull request by valdelaseras**
... **in progress still** :) But have a look in the meantime! 

I realise its a lot so take your time and don't hesitate to ask any questions as to how/why or what not. 

this branch contains in short: 

- restructuring and renaming of your directories and file names
- restructure and update your CSS ( main point being: we really don't want different style sheets for different pages etc. We want one system that works for all of our pages and all screen sizes etc. )
- my starting point is blog.html, I have not updated the templates of the other pages yet. Once the updated CSS is done, updating the pages templates is a piece of cake. If you want, you can try to do it yourself to get a better understanding of this particular system

### CODE REVIEW

**Comment by valdelaseras** [comment]
Screenshot 2022-11-21 at 3 18 00 PM

A general design tip:

Aligning content can really help give your website a clean and well organised look. Have a look at these 2 boxes: you have the width of the `<article>` ( where your entries are, the green box ) and the width of the content within the article ( red box ). Your footer content width is in between these 2. A little handy tip: pick either one of the green or red box widths instead for your footer content width. 

In this case I choose the green box personally but its up to you what you prefer ofc. 

Screenshot 2022-11-21 at 3 23 13 PM

**Comment by valdelaseras** [comment]
Screenshot 2022-11-21 at 4 30 42 PM

^ in this way, no matter how many ( or little ) nav links you add, it will always be responsive and functional. It's maybe not the most super aesthetically pleasing nav, but it doesn't look glitchy and it *works*, that is very / most important. 

On a more UX note: the nav font is not very readable like this. We need to come up with something better

**Comment by valdelaseras** [comment]
Screenshot 2022-11-21 at 4 44 28 PM


here is another example of "invisible boxes and lines" we use to align content. Align the top branding logo with the fixed theme toggle

**Comment by valdelaseras** [comment]
Screenshot 2022-11-21 at 4 58 09 PM

I would opt for this kind of placement I think re the alignment and paddings of the "corner elements" ( logo, nav list, theme toggle ). 20px on each side, as the default padding now is across all content blocks ( 20px ). 

it creates cohesion

File: styles/components/theme-toggle.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+.theme-toggle p {
+    line-height: 0;
+    color: var(--footer-text-secondary);
+}
+
+@media only screen and (min-width: 1440px) {
+    .theme-toggle {
+        display: inline-block;
+    }
+}
```
Comment (valdelaseras) [inline styles/components/theme-toggle.css]: relevant note re theme switch

File: styles/themes/colors.css [hunk 2]
```diff
@@ -28,12 +28,10 @@
...
-  --footer: #292c2f;
-  --footer-white: #f5f5f5;
-  --footer-icons: #b194a8;
-  --footer-name: #a5a9af;
-  --footer-link: #5383d3;
-  --footer-about: #92999f;
+  --footer-text-primary: #a5a9af;
+  --footer-text-secondary: #f5f5f5;
+  --footer-bg: #292c2f;
+  --footer-icon: #b194a8;
```
Comment (valdelaseras) [inline styles/themes/colors.css:36]: I would strongly consider to reduce the amount of colours. A lot of them are very similar but vary only slightly. Pick 1 kind of white for instance for the footer text and the article background etc. 

Usually I go with a colour setup that looks like this ( picking random colours for demo purpose ): 

```css
:root {
    /* base colors */
    --primary-color: white;
    --secondary-color: black;
    --tertiary-color: grey;
    
   /* accent colors */
    --primary-accent-color: pink;
    --secondary-accent-color: lightpink;
    
    /* fonts */
    --primary-font-color: var(--primary-color);
    --secondary-font-color: var(--tertiary-color);
    
    --primary-accent-font-color: var(--primary-accent-color);
    --secondary-accent-font-color: var(--secondary-accent-color);
}
```

Note here how font colors use the base and accent colours we defined. For your footer for instance, you would then just say: 

```css
.footer {
   color: var(--secondary-font-color);
}

.footer-link-list {
    color: var(--primary-font-color);
}
```

etc. This way, when you decide you want to update 1 shade of color ( for ex your greys ), you only need to change the hexcode in 1 line. 

Using a limited number of colors also brings cohesion to your site, much like with the alignments / invisible boxes I explained before. I think 5, max 8 base colours is good

File: styles/layout/column.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
+@media only screen and ( min-width: 1440px ) {
```
Comment (valdelaseras) [inline styles/layout/column.css:1]: I've chosen the breakpoint between the mobile and desktop view to be 1440 btw, but you could also pick 1024 if you'd want. 768 becomes a bit risky already, so I wouldn't go lower than 1024 but its up to you to check it it out.

If you want to change it, be sure to CTR + F 1440 across all CSS files and adjust all of them to 1024 in that case

**Comment by KyootyBella** [comment]
Merched into dev, to start working further on the refactoring and get to v1.0 of the site

**Pull request merged**